### ISSUE TITLE
Standardise colors

### ISSUE THREAD

**Issue by jameskoster** [woocommerce/woocommerce-admin#3897]
TLDR - Remove any style overrides applied to components we're consuming from the WordPress core component package so that we gracefully inherit "[G2](https://www.figma.com/file/fnyj380i05vGzuuH60frLQ/G2-Design?node-id=85%3A6975)" design updates. https://github.com/WordPress/gutenberg/issues/18667

To make WooCommerce feel like a seamless part of the broader WordPress UI, and to better respect user preferences we intend to remove unnecessary applications of color, and be more intentional with our branding. More info in p6riRB-5na

Here's a list of elements I found that will need updating:

# Profiler
* [Step indicator backgrounds](https://cloudup.com/cLHdei5fl0a) should be `$blue-medium-focus` instead of purple.
* Focussed input border should match Gutenberg (`$blue-medium-focus`).
* Elevated elements (state / country dropdown, Digalogs) should use `$shadow-popover` for shadow styling.
* Tab styling on the theme step should inherit default Tab component colors.

# Task list
* Purple icons in the task list should be `$blue-medium-focus`.
* Snackbar styling and positioning should match Gutenberg.
* Image upload button (add a logo task) should match the button in the Image block (https://cloudup.com/cmauCa__tYQ).

# Activity Panel
* Orange notification indicators should use `$alert-red`.
* Tabs should inherit the default colorisation. No purple stroke on active tabs. Icon colors should match text.
* Set side sheet background to `$light-gray-200`.
* Set header background to `$light-gray-500`.
* Error icon in the inbox should be `$alert-yellow` instead of purple.

# Analytics
* "Presets" / "Custom" button group in the date picker should inherit the default button colors.
* The square that indicates an active selection in the date-picker should be `$blue-medium-focus`.
* The date-picker dropdown select colors should match core. Reference: https://cloudup.com/c1u1rQBTec3 
* Cards should ideally be updated to use the [`Card`](https://wordpress.github.io/gutenberg/?path=/story/components-card--default) component. Appreciate that's a big job though. If we can at least ensure the border colors match (use `$light-gray-500`) that'd be great!
* When Summary Numbers are links, the stroke at the top of the active state should be `$blue-medium-focus` instead of purple.
* The bar/link chart switcher buttons should match Gutenberg editor toolbar buttons. E.g. https://cloudup.com/cZFdWiYumvC
* The "by day, by week, by month" dropdown in charts should inherit core styles. Same as the date-picker.
* The "Download" button in report tables should have standard button styling.
* Search inputs should match the search input in the block inserter - https://cloudup.com/cRHPDpdGyUL

# General
* All buttons should inherit default colorisation.
* All text links should inherit the default blue color.
* Default text color should be `$medium-gray-text`.
* Headings should be `$dark-gray-900`.
* Checkbox / Switch toggle active states should inherit core colorisation (blue) rather than purple.
* Icon buttons should inherit color colorisation.
* All grey strokes should be `$light-gray-500`.

All colors vars pulled from https://github.com/WordPress/gutenberg/blob/master/packages/base-styles/_colors.scss

Any questions? Please ask!

cc @LevinMedia in case I missed anything.

**Comment by psealock** [comment]
>Task List

The quick links are `$theme-color`, would these be as well?

![Screen Shot 2020-06-15 at 3 53 28 PM](https://user-images.githubusercontent.com/1922453/84616680-c8267800-af20-11ea-87a4-d994106b3f2f.png)

**Comment by jameskoster** [comment]
Closing in favor of https://github.com/woocommerce/woocommerce-admin/issues/4732
